features/step_definitions/node.rb

```diff
@@ -603,6 +603,11 @@
   raise "Error syncing files over to '#{pod.name}' pod '#{@result[:stderr]}'" unless @result[:success]
 end
 
+Given /^I store all master nodes to the#{OPT_SYM} clipboard$/ do |cb_name|
```

Comment:
We have `Given /^I store the( schedulable| ready and schedulable)? (node|master|worker)s in the#{OPT_SYM} clipboard$/ do |state, role, cbname|`

Reply:
ok, didn't know that existed.  I'll just close the PR then.